database-constants.js

```diff
@@ -3,7 +3,9 @@ module.exports = Object.freeze({
     DATABASE_NAME: 'crdc-datahub',
     SESSION_COLLECTION: 'sessions',
     APPLICATION_COLLECTION: 'applications',
+    SUBMISSION_COLLECTION: 'submissions',
     USER_COLLECTION: 'users',
     ORGANIZATION_COLLECTION: 'organization',
-    LOG_COLLECTION: 'logs'
+    LOG_COLLECTION: 'logs',
+    API_TOKEN: 'api-token'
```

Comment:
API_TOKEN has been defined in user-constants.js, can we reuse that instead of define it here?